src/otx/core/data/dataset/action_detection.py

```diff
@@ -45,7 +45,6 @@ def _get_item_impl(self, idx: int) -> ActionDetDataEntity | None:
                 img_idx=idx,
                 img_shape=img_shape,
                 ori_shape=img_shape,
-                pad_shape=img_shape,
                 scale_factor=(1.0, 1.0),
```

Comment:
Does action detection require scale_factor?

Reply:
Why not? Is the action detection has images?

Reply:
I mean all the other algo removed scale_factor attribute, but action detection still takes scale_factor as an input.

Reply:
Fixed in 0b833ae4920894981f161eacaac0c791acca9b27